### ISSUE TITLE
Incorrect merging of conflicting argument values

### ISSUE THREAD

**Issue by cdesaintguilhem** [WGUNDERWOOD/tex-fmt#73]
With the following config file at `/Users/<username>/Library/Application\ Support/tex-fmt/tex-fmt.toml`:
```toml
# Cyprien's tex-fmt.toml
check = false
print = false
wrap = true
wraplen = 120
tabsize = 2
tabchar = "space"
stdin = false
verbosity = "info"
```
running `tex-fit --args` from `~/` yields the following output:
```shell
tex-fmt
  check:       false
  print:       false
  wrap:        true
  wraplen:     80
  wrapmin:     70
  tabsize:     2
  tabchar:     space
  stdin:       false
  config:      /Users/<username>/Library/Application Support/tex-fmt/tex-fmt.toml
  verbosity:   warn
```

This shows that the configuration file is correctly picked up from the default macOS location, but the merge resolution of `120` with `80` yields the default value of `80` rather than the expected `120`. Similarly, the merge resolution of `info` with `warn` yields the default of `warn` rather than the expected `info`.

**Comment by WGUNDERWOOD** [comment]
Thanks for letting me know, I'll try to look into this soon.

**Comment by WGUNDERWOOD** [comment]
Should be fixed in fcc4545ecbe3192b7cb3dbaa0bfc2fd2e8b72c91. The issue was using default values in clap takes precedence over struct merging.

**Comment by cdesaintguilhem** [comment]
Thanks for taking the time! After installing with cargo from fcc4545, I get the following output:
```zshell
❯ tex-fmt --args
tex-fmt
  check:       false
  print:       false
  wrap:        true
  wraplen:     120
  wrapmin:     110
  tabsize:     2
  tabchar:     space
  stdin:       false
  config:      /Users/username/Library/Application Support/tex-fmt/tex-fmt.toml
  verbosity:   warn

~ 
❯ cat /Users/username/Library/Application\ Support/tex-fmt/tex-fmt.toml
# Cyprien's tex-fmt.toml
check = false
print = false
wrap = true
wraplen = 120
tabsize = 2
tabchar = "space"
stdin = false
verbosity = "info"
```
I see that the `wraplen` argument is correctly merged, but not the `verbosity` one.

**Comment by WGUNDERWOOD** [comment]
This should be fixed in 910db8f3be8794e5c547cd7d184cf8104c5081a5. The problem was I had not implemented `verbosity = "info"`, and was instead expecting `verbosity = "verbose"`. Now either of these should work.

**Comment by WGUNDERWOOD** [comment]
Thanks for this, should be fixed in 5406ca6914efc3c5d05d5ae7118a9e8308cda152.
